### PR TITLE
feat(icon): add icon component

### PR DESCRIPTION
**Pull request by dine**

### CODE REVIEW

File: src/components/ui-notification.vue [hunk 1]
```diff
@@ -1,110 +1,40 @@
...
+                class="text-yellow-1"
+            ></ui-icon>
         </span>
 
         <slot name="notificationBody" />
     </div>
 </template>
 
 <script>
+import UiIcon from './ui-icon.vue';
```
Comment (larsdouweschuitema) [inline src/components/ui-notification.vue:34]: Are we still using aliases? Although this path is rather simple, it makes it cleaner.

File: src/stories/ui-icon.stories.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+export default {
+    title: 'Informational/Icon',
+    component: UiIcon,
+    argTypes: {
+        icon: {
+            control: {
+                type: 'select',
+                options: [
+                    'addHouse',
```
Comment (larsdouweschuitema) [inline src/stories/ui-icon.stories.js:11]: Can we keep the export itself clean and move this inside a const?

File: src/stories/ui-notification.stories.js [hunk 3]
```diff
@@ -29,19 +30,11 @@ const Template = (args, { argTypes }) => ({
 
                 <button
                     type="button"
-                    class="leading-6 rounded-sm px-6 h-11 inline-flex text-center items-center justify-center align-middle cursor-pointer whitespace-no-wrap ml-3 bg-transparent border-none p-0 h-auto border-0 text-dark-2"
+                    class="leading-6 rounded-sm inline-flex text-center items-center justify-center align-middle cursor-pointer whitespace-no-wrap ml-3 bg-transparent border-none p-0 h-auto border-0 text-dark-2"
```
Comment (larsdouweschuitema) [inline src/stories/ui-notification.stories.js:33]: Achievement unlocked: LONG CLASS NAME 🥇

I guess this is the downside of not using one class and assigning styles, but trying to style everything using ui classes.

**Review by jpsc — APPROVED** [review]

File: src/components/ui-notification.vue (hunk 1 above)
Comment (dine) [reply src/components/ui-notification.vue:34]: Using the @ alias here will break when we import this components into another project (that has different path for @).

**Review by bartderijk — DISMISSED** [review]

**Pull request merged**